Week1/homework/js-exercises/exercise5.js

```diff
@@ -0,0 +1,17 @@
+// exercise 5
+"use strict"
+let str="";
+function print(str) {
+    console.log(str);
+}
+let z=7.25;
```

Comment:
Tip: in programming we try to give "meaningful" variable names, so it's easier to understand what is going on, so for instance.

```
let firstNumber = 7.25
```

This gives some more insights on someone else that is reading the code, and makes it easier to understand what is going on! 

Furthermore everything seems to be working fine, so make an extra effort on these details that really make a big difference!